README.md

```diff
@@ -1 +1,215 @@
+
+In this step you are performing the role of the data owner, registering his data in the data catalog and registering the credentials for accessing the data in the credential manager.
+
+We now explain how to register a dataset in the Katalog data catalog.
+
+Begin by registering the credentials required for accessing the dataset as a kubernetes secret. Replace the values for `access_key` and `secret_key` with the values from the object storage service that you used and run:
+
+```bash
+cat << EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+    name: paysim-csv
+type: Opaque
+stringData:
+    access_key: "${ACCESS_KEY}"
+    secret_key: "${SECRET_KEY}"
+EOF
+```
+
+Next, register the data asset itself in the data catalog.
+We use port-forwarding to send asset creation requests to the Katalog connector.
+
+```bash
+cat << EOF | kubectl apply -f -
+apiVersion: katalog.fybrik.io/v1alpha1
+kind: Asset
+metadata:
+  name: paysim-csv
+spec:
+  secretRef: 
+    name: paysim-csv
+  details:
+    dataFormat: csv
+    connection:
+      name: s3
+      s3:
+        endpoint: "http://localstack.fybrik-notebook-sample.svc.cluster.local:4566"
+        bucket: "demo"
+        object_key: "PS_20174392719_1491204439457_log.csv"
+  metadata:
+    name: Synthetic Financial Datasets For Fraud Detection
+    geography: theshire 
+    tags:
+      finance: true
+EOF
+```
+
+### Define data access policy
+
+Acting as the data steward, define an [OpenPolicyAgent](https://www.openpolicyagent.org/). In this example we just access the data asset without redacting columns. Below is the policy (written in [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/#what-is-rego) language):
+
+```rego
+package dataapi.authz
+rule[{}] { true }
+```
+In this sample only the policy above is applied. Copy the policy to a file named `sample-policy.rego` and then run:
+
+```bash
+kubectl -n fybrik-system create configmap sample-policy --from-file=sample-policy.rego
+kubectl -n fybrik-system label configmap sample-policy openpolicyagent.org/policy=rego
+while [[ $(kubectl get cm sample-policy -n fybrik-system -o 'jsonpath={.metadata.annotations.openpolicyagent\.org/policy-status}') != '{"status":"ok"}' ]]; do echo "waiting for policy to be applied" && sleep 5; done
+```
+
+### Deploy a Fybrik application
+
+Create a `FybrikApplication` resource to register the notebook workload to the control plane of Fybrik. The value you place in the `dataSetID` field is your asset ID, as explained above. you can run the following to Create a `FybrikApplication` resource for this example:
+
+```yaml
+cat <<EOF | kubectl apply -f -
+apiVersion: app.fybrik.io/v1beta1
+kind: FybrikApplication
+metadata:
+  name: my-notebook
+  labels:
+    app: my-notebook
+spec:
+  selector:
+    workloadSelector:
+      matchLabels:
+        app: my-notebook
+  appInfo:
+    intent: Fraud Detection
+  data:
+    - dataSetID: "fybrik-notebook-sample/paysim-csv"
+      requirements:
```

Comment:
add a missing "read" capability to the requirements.

Reply:
I think you mean `flow` field under `data`? according to the [API reference](https://fybrik.io/v1.3/reference/crds/#fybrikapplicationspecdataindex) it seems that if we want to indicate what is being done with the particular dataset it should be under `flow` but it assumes default read.
I can add that for readability if you think that is better

Reply:
as we discussed, I added flow field under data and put read as the flow